### PR TITLE
main/zsh: update to 5.7.1

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by J0WI** [comment]
SourceForge seems not to be a reliable mirror...

**Comment by 5paceToast** [comment]
@J0WI can you rebase this?

**Comment by Ikke** [comment]
@J0WI Could you rebase this to fix CI?

**Comment by J0WI** [comment]
Done.

File: main/zsh/APKBUILD [hunk 1]
```diff
@@ -8,10 +8,10 @@
...
 #
 pkgname=zsh
-pkgver=5.6.2
+pkgver=5.7.1
 pkgrel=0
 pkgdesc="Very advanced and programmable command interpreter (shell)"
-url="http://www.zsh.org"
+url="https://www.zsh.org"
 arch="all"
 license="BSD"
```
Comment (maxice8) [inline main/zsh/APKBUILD]: license= is not an SPDX identifier and is ambiguous

Comment (J0WI) [reply main/zsh/APKBUILD]: Rebased

**Comment by J0WI** [comment]
Upgrade fixed in af2c3849211f5b0c24e2554e5b9ee07a4ad66d03 and license in 4915451a022511db466f97826f675eaba0bb1ccd

**Pull request closed without merging**